components/avatar/avatar.vue

```diff
@@ -5,7 +5,7 @@
     :class="avatarClasses"
     data-qa="dt-avatar"
     :aria-label="buttonAriaLabel"
-    @click="handleClick"
+    @click.stop="handleClick"
```

Comment:
What was the case here?

Reply:
The event was being emitted twice on tests, the avatar click and the callbox click